programs/ssl/ssl_client2.c

```diff
@@ -957,7 +914,7 @@ int main(int argc, char *argv[])
     opt.groups              = DFL_GROUPS;
     opt.sig_algs            = DFL_SIG_ALGS;
 #if defined(MBEDTLS_SSL_EARLY_DATA)
-    opt.early_data_file     = DFL_EARLY_DATA_FILE;
+    opt.early_data          = DFL_EARLY_DATA_FILE;
```

Comment:
```suggestion
    opt.early_data          = DFL_EARLY_DATA;
```

Reply:
Provide two options, both DFL_EARLY_DATA_FILE and DFL_EARLY_DATA are exist, please help review again.